staging/src/k8s.io/apiserver/pkg/storage/cacher/cacher.go

```diff
@@ -68,8 +68,8 @@ type Config struct {
 	// An underlying storage.Versioner.
 	Versioner storage.Versioner
 
-	// The Cache will be caching objects of a given Type and assumes that they
-	// are all stored under ResourcePrefix directory in the underlying database.
+	// ResourcePrefix directory stores the empty object created by NewFunc function 
```

Comment:
This comment doesn't make any sense to me.

ResourcePrefix is a prefix that is attached to each key of items stored in etcd (underlying DB).

Reply:
The concept of `Type` was replaced by a function that creates the empty objects, so I changed the lines